### PR TITLE
Warning login

### PR DESCRIPTION
**Pull request by AntoineMarnat**
See https://forum.camptocamp.org/t/nouvelle-interface-bugs-questions-suggestions/241148/851

Adds a warning that login and password are case-senvitive.

### CODE REVIEW

File: src/views/user/LoginView.vue [hunk 1]
```diff
@@ -378,4 +381,10 @@ export default {
     padding: 1.5rem;
   }
 }
+.login-view-warning{
+  align-items: center;
+  text-align: center;
+  margin-bottom: 1em;
+  //color:red ;
```
Comment (ataraxido) [inline src/views/user/LoginView.vue:388]: import color variable ?

File: src/views/user/LoginView.vue [hunk 2]
```diff
@@ -22,6 +22,9 @@
         :label="$gettext('Password')"
         icon="key"
       />
+      <div class="login-view-warning" v-translate>
+        <b> Beware of case-sensitive login and password</b>
```
Comment (ataraxido) [inline src/views/user/LoginView.vue:26]: leading space ?

Comment (ataraxido) [inline src/views/user/LoginView.vue:26]: bold as a div style class ?

File: src/views/document/RouteView.vue [hunk 3]
```diff
@@ -77,7 +77,12 @@
 
         <div class="box">
           <markdown-section :document="document" :field="fields.summary" />
-          <markdown-section :document="document" :field="fields.route_history" />
+          <markdown-section v-if="locale.route_history" :document="document" :field="fields.route_history" />
+          <div v-else-if="showMissingHistoryBanner" class="notification is-info no-print">
+            <edit-link :document="document" :lang="lang" show-even-if-not-logged>
+              History is missing, please provide it if you have information.
+            </edit-link>
+          </div>
```
Comment (ataraxido) [inline src/views/document/RouteView.vue:85]: wrong/mixed commit ?

File: src/views/user/LoginView.vue (hunk 2 above)
Comment (cbeauchesne) [reply src/views/user/LoginView.vue:26]: Do not use `<b>`, prefer `<span class="has-text-weight-bold">`

It seems way more complex, yes :)

File: src/views/document/RouteView.vue (hunk 3 above)
Comment (cbeauchesne) [reply src/views/document/RouteView.vue:85]: probably :)

File: src/views/user/LoginView.vue [hunk 4]
```diff
@@ -378,4 +381,10 @@ export default {
     padding: 1.5rem;
   }
 }
+.login-view-warning{
+  align-items: center;
+  text-align: center;
+  margin-bottom: 1em;
```
Comment (ataraxido) [inline src/views/user/LoginView.vue:387]: class mb-1 ?

**Review by ataraxido — CHANGES_REQUESTED** [review]

File: src/views/document/RouteView.vue [hunk 5]
```diff
@@ -180,6 +180,27 @@ export default {
...
+      for (let act of history_worth_activities) {
+        if (activities.includes(act)) {
+          return true;
+        }
+      }
+      if (activities.includes('skitouring') && ['5.1', '5.2', '5.3', '5.4', '5.5'].includes(doc.ski_rating)) {
+        return true;
+      }
+      return false;
+    },
```
Comment (ataraxido) [inline src/views/document/RouteView.vue:203]: wrong commit

**Review by ataraxido — CHANGES_REQUESTED** [review]

**Pull request closed without merging**